index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio: Setup and Mobile First</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;400;600;700&display=swap">
+</head>
+<body>
+  <div class="container">
```

Comment:
- [ ] Please for semantics purpose, kindly use the `header` tag in place of the div tag. The [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) header element represents introductory content, typically a group of introductory or navigational aids. 👍